.storybook/config.js

```diff
@@ -11,6 +11,7 @@ function loadStories() {
   // add any topic stories here!
   require('../app/stories')
   require('../agents/stories')
+  require('../offerings/stories')
```

Comment:
i wonder if the topic should be "supply" or "suppliers"? open to how granular or coarse we want our topics to be.